main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
Versioning should be synced with application version otherwise it will be tricky to keep track of this. I would suggest to use stuff from `version.go` as it's autoset by build system so when we set tags or branch in git, mongo version would moved automatically.